modules/ROOT/pages/install-azure.adoc

```diff
@@ -32,50 +48,35 @@ On Windows, you must have a shell terminal emulator (such as cygwin) or access t
 
 . Locate your organization's Mule Enterprise license key file (`license.lic`) and transfer the file to your Unix environment if necessary.
 . Open a terminal/shell and navigate to the `azure` sub-directory.
-. Copy the following to a text editor, then run:
+. Copy the following to a text editor:
 +
 ----
 RTF_MULE_LICENSE='' \
 ./generate-templates.sh
 ----
 
-. Using the terminal, encode the Mule Enterprise license key to base64 and paste the contents as the value for `RTF_MULE_LICENSE` in the text editor.
+. Using the terminal, encode the Mule Enterprise license key to base64 and paste the contents as the value for `RTF_MULE_LICENSE` in your text editor.
 +
 ----
 base64 -w0 license.lic # Linux
-base64 -b0 license.lic # OSX
+
+base64 -b0 license.lic # OSX / MacOS
 ----
 
 . Copy the command from the text editor and run in the terminal pointed to the `azure` directory.
-. Confirm the output of the `ARM-template-dev.json` and `ARM-template-prod.json` files in the `azure` directory.
-
-== Installation
-
-To begin installing Runtime Fabric, download the Azure Resource Manager template from Runtime Manager
-
-. From Anypoint platform, navigate to Runtime Manager.
-. Select *Runtime Fabrics*.
-. Click on the Runtime Fabric you created. It should be in the `Activating` state.
-. Click the *Download files* link.
-. After the file has downloaded, unzip the `rtf-install-scripts.zip` file.
-
-The `rtf-install-scripts\azure` directory contains the following installation files:
-
-* `ARM-template-dev.template`: the Azure Resource Manager template that describes the infrastructure for the development configuration requirements.
-* `ARM-template-prod.template`: the Azure Resource Manager template that describes the infrastructure for the production configuration requirements.
-* `generate-templates.sh`: a shell script used to insert the Mule Enterprise license digest and initialization script in the ARM templates. This script outputs the JSON templates to use when deploying to Azure.
+. Confirm the generated output of the `ARM-template-dev.json` and `ARM-template-prod.json` files in the `azure` directory.
 
 == Deploy using the ARM Template
 
-Run the ARM template to provision the infrastructure on your Azure account. The following procedures describe how to run the template via the Azure Portal.
+Run the generated ARM template to provision the infrastructure on your Azure account. The following procedures describe how to run the template via the Azure Portal.
 
 [NOTE]
 You must have a private and public key as a PEM file to provision the VMs. This is required to enable secure access to your VMs via SSH (Secure Shell).
 
 . Log into your https://portal.azure.com[Azure portal].
 . Navigate to *Create a resource*.
 . From the *Search the marketplace* search bar, enter *Template deployment*.
-. Click *Create*.
+. Click *Create* on the bottom of the window.
```

Comment:
@sundippatel I try to avoid mentioning location of elements within the UI. UIs tend to change without notifying us. If our UI is that complicated, we have other issues. ;-)

Reply:
👍